src/lib/stores/marketplace.ts

```diff
@@ -0,0 +1,686 @@
+export const marketplace = [
+    {
+        icon: 'icon-lightning-bolt',
+        id: 'starter',
+        name: 'Starter',
+        tagline: 'A simple function. Create something awesome! 🚀',
```

Comment:
```suggestion
        tagline: 'A simple function to get started. Edit this function to explore endless possibilities with Appwrite Functions. 🚀',
```

@Meldiron can we also add a link to docs in these descriptions?

[Learn more about functions](/docs/functions).